src/ui/components/SecondaryToolbox/react-devtools/ReactDevToolsListData.ts

```diff
@@ -0,0 +1,92 @@
+import assert from "assert";
```

Comment:
:hand: intended to be an import of one of our own utils instead?

Reply:
No, I meant to import this one. We also have our own `assert()` functions (a few of them). They're all equivalent I think. I'm in the habit of importing this one though 🤷🏼